packages/main/src/plugin/kubernetes/backoff.spec.ts

```diff
@@ -21,7 +21,7 @@ import { expect, test } from 'vitest';
 import { Backoff } from './backoff.js';
 
 test('backoff increment and limit without jitter', () => {
-  const backoff = new Backoff(100, 500, 0);
+  const backoff = new Backoff(100, 2, 500, 0);
```

Comment:
if possible, move to an object as parameter with default value

b/c reading new Backoff(100, 2, 500, 0) it's hard to know what is doing what

so either move 100 to BACKOFF_ VALUE, 2 to BACKOFF_MULTIPLIER

or introduce `new Backoff({value: 100, multiplier: 2, max: 500}) (and then jitter has an optional/default value of 0 anyway)